.github/workflows/blackduck.yml

```diff
@@ -1,9 +1,8 @@
 name: blackduck-scan
 
 on:
-  pull_request: ~
-  push:
-    branches: ['main']
+  schedule:
```

Comment:
Maybe we can add a manual trigger so we can make a scan before a release.

Reply:
You can find the trigger in the release pipeline.

Reply:
Ah good idea.

Reply:
✅